pkg/reconciler/kubernetes/tektonchain/tektonchain.go

```diff
@@ -680,44 +677,33 @@ func AddControllerEnv(controllerEnvs []corev1.EnvVar) mf.Transformer {
 
 func GenerateSigningSecrets(ctx context.Context) map[string][]byte {
```

Comment:
looks like we dont need to export this: s/GenerateSigningSecrets/generateSigningSecrets

Reply:
+1